### ISSUE TITLE
Bad on iOS

### ISSUE THREAD

**Issue by qiujie8092916** [stackblitz/use-stick-to-bottom#9]
I found the scrolling animation on my iphone is so slow and  <code>escapedFromLock </code> parameter is incorrectly determined.

ios: 18.1.1
broswer: chrome

**Comment by samdenty** [comment]
yep since bolt.new doesn't target iOS (safari is like the new IE) i haven't yet tweaked all the stuff to work correctly on iOS. I'm down to doing it though (or someone sending a PR), but there's a load of stuff you have to test to ensure everything works correctly. I must have done like 1000 commits to get it to work this well on desktop

**Comment by samdenty** [comment]
yep since bolt.new doesn't target iOS i haven't yet tweaked all the stuff to work correctly on iOS. I'm down to doing it though (or someone sending a PR), but there's a load of stuff you have to test to ensure everything works correctly. I must have done like 1000 commits to get it working well on desktop for bolt.new, so i wouldn't want to break anything
